src/pages/Bridge/RedeemForm/index.tsx

```diff
@@ -304,17 +304,13 @@ const RedeemForm = (): JSX.Element | null => {
           }
         }
 
-        // FIXME: workaround to make premium redeem still possible
-        const relevantVaults = new Map<InterbtcPrimitivesVaultId, BitcoinAmount>();
-        // FIXME: a bit of a dirty workaround with the capacity
-        relevantVaults.set(vaultId, monetaryWrappedTokenAmount.mul(2));
         const result = await window.bridge.redeem.request(monetaryWrappedTokenAmount, data[BTC_ADDRESS], vaultId);
 
         // TODO: handle redeem aggregator
         const redeemRequest = result[0];
         handleSubmittedRequestModalOpen(redeemRequest);
         setSubmitStatus(STATUSES.RESOLVED);
-      } catch (error) {
+      } catch (error: any) {
```

Comment:
It seems like it's unrelated to the purpose of this PR. I wonder if you were using TypeScript version installed in your machine instead of in the project node_modules. In that case, there might have been TypeScript error warning on this line. Please make sure that you use the TypeScript version installed in the node_modules.